### DISCUSSION TITLE
Tab Groups, Servo and AppImage

### DISCUSSION THREAD

**Discussion by maverick74** [Floorp-Projects/Floorp#691] [Feature Request]
This seems great, but i would like to ask a few questions/suggest some features:

Any plans to support **Containerized Tab Groups** (see: https://addons.mozilla.org/en-US/firefox/addon/container-tab-groups/ ) natively / Out Of The Box?

Any possibility to eventually (long term) allow the Switch-to / use of Servo (servo.org) as an alternative rendering engine???  _yeap - that's a veeery looong shot!!!_

What about AppImage? Any plans to distribute Floorp in AppImage format?

**Comment by summoner001** [comment]
Yeah tab grouping is a good idea!

**Comment by CutterKnife** [comment]
The tab group was considered many times but was abandoned due to technical problems.
If you simply want to separate tabs for each container, there are features similar to Workspaces. Please give it a try!
